.github/workflows/ci-fmudataio.yml

```diff
@@ -42,7 +42,12 @@ jobs:
             "fmu-sumo-uploader @ git+https://github.com/equinor/fmu-sumo-uploader.git"
 
       - name: Full test
-        run: pytest -v --cov --cov-report term-missing
+        run: |
+          if [[ "${{ github.event.pull_request.base.ref }}" == "staging" ]]; then
+            pytest -n auto -v --cov --cov-report term-missing --prod
```

Comment:
Both pytest and update-schemas take a `--prod` flag now. Both ensure the `DEV_SCHEMAS` environment variable is unset. This causes the default case to happen: schemas and metadata are produced with the production url.